addons/sales_team/models/crm_team_member.py

```diff
@@ -131,10 +131,11 @@ def _compute_member_warning(self):
                 teams = user_mapping.get(member.user_id, self.env['crm.team'])
                 remaining = teams - (member.crm_team_id | member._origin.crm_team_id)
                 if remaining:
-                    member.member_warning = _("Adding %(user_name)s in this team would remove him/her from its current teams %(team_names)s.",
-                                              user_name=member.user_id.name,
-                                              team_names=", ".join(remaining.mapped('name'))
-                                             )
+                    member_warning = _("Adding %(user_name)s in this team will remove them from %(team_names)s.",
+                                       user_name=member.user_id.name,
+                                       team_names=", ".join(remaining.mapped('name'))
+                                      )
+                    member.member_warning = member_warning + " " + _("Working in multiple teams? Activate the option under Configuration>Settings.")
```

Comment:
Can't we do it using a single string ?

Reply:
@tde-banana-odoo 
If I do it like this:
```python
member.member_warning = _("Adding %(user_name)s in this team will remove them from %(team_names)s. \
                            Working in multiple teams? Activate the option under Configuration>Settings.",
                            user_name=member.user_id.name,
                            team_names=", ".join(remaining.mapped('name'))
                          )
 ```
 The `\` breaks the line and the alert looks like this on the display:
 
![image](https://github.com/user-attachments/assets/b653c668-10a1-4c61-bfed-b8dfe1fba056)

I could also set the whole sentence on a single line but it will be way too long.
What do you think?

Reply:
Probably because you make a multi line string while not necessary ^^ you can close and begin at next line
```
    _("Adding %(user_name)s in this team will remove them from %(team_names)s. "
       "Working in multiple teams? Activate the option under Configuration>Settings.",
       user_name=member.user_id.name,
       team_names=", ".join(remaining.mapped('name'))
   )
```

Reply:
@tde-banana-odoo 
I learned something :smile: 
Thanks!